### PR TITLE
Add GitHub Actions integration

### PR DESCRIPTION
**Pull request by mosuka**
Add GitHub Actions integration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,6 @@ UniDic builder for [Lindera](https://github.com/lindera-morphology/lindera). Thi
 The following products are required to build:
 
 - Rust >= 1.46.0
```
Comment (johtani) [inline README.md:17]: Should we have a notification about a supported lindera version?

`Lindera >= 0.6.0` or something like this?

**Review by johtani — APPROVED** [review]
Basically, LGTM. One minor comment to README.md

File: README.md (hunk 1 above)
Comment (mosuka) [reply README.md:17]: I don't need to write it down as we plan to have users use it with a match to a major and minor version number in the future.
If I get a lot of inquiries, I'll think about it.

Comment (johtani) [reply README.md:17]: I see. It's OK :)

**Pull request merged**